src/plugins/mvcc_delete_plugin.cpp

```diff
@@ -76,9 +80,18 @@ void MvccDeletePlugin::_logical_delete_loop() {
 
           std::unique_lock<std::mutex> lock(_mutex_physical_delete_queue);
           _physical_delete_queue.emplace(table, chunk_id);
+          saved_memory += chunk_memory;
+          num_chunks++;
         }
       }
     }
+    if (saved_memory > 0) {
+      std::ostringstream message;
+      double saved_mb = std::round(static_cast<float>(saved_memory) / (10000.0)) / 100.0;
```

Comment:
The intention here is only to scale bytes to megabytes? I would find dividing once by `1000000.0` or twice by `1000.0` easier to understand.

Reply:
I think the idea is to round it to two decimal places, but std::setprecision would be the better approach.